.buildkite/pipeline.yml

```diff
@@ -91,6 +99,21 @@ steps:
       distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
       hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
 
+  - label: ':hammer: test against firecracker master'
+    commands:
+      - 'ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux'
+      - 'ln -s /var/lib/fc-ci/rootfs.ext4 testdata/root-drive.img'
+      - 'cp /usr/local/bin/firecracker-v0.20.0 testdata/firecracker'
+      - 'cp /usr/local/bin/jailer-v0.20.0 testdata/jailer'
```

Comment:
It is confusing to see cp 0.20.0 under master test. Do we need them?

Reply:
Fixed